### ISSUE TITLE
repository url in package.json is formatted weirdly?

### ISSUE THREAD

**Issue by thunfisch987** [victorgarciaesgi/nuxt-typed-router#138]
**Describe the bug**

(i'm currently building and testing a small script for myself, which outputs the repositories when updating the packages, so i can look at the changelogs)
I noticed the repository url in the package.json is formatted weirdly?
```json
{
  "repository": {
    "type": "git",
    "url": "git+https://victorgarciaesgi@github.com/victorgarciaesgi/nuxt-typed-router.git"
  },
}
```
I do not mean the `git+`  but the `https://...@github.com/...`, why the `...@`?

**Expected behavior**

\-

**Screenshots**

\-

**Environnement infos**

Run `nuxi infos`

-

**Your `pages` folder structure**

Run `npx tree-node-cli {your page folder path}`

-

**Your nuxt.config.ts**
Paste your modules config (and i18n config if you're using it)

-

**Comment by victorgarciaesgi** [comment]
When I created it I though it was required, i'll check! :)

**Comment by victorgarciaesgi** [comment]
Can confirm everybody does this, even Vue: https://github.com/vuejs/core/blob/c3087ff2cce7d96c60a870f8233441311ab4dfb4/packages/vue/package.json#L86C1-L86C1

**Comment by thunfisch987** [comment]
sorry, i do _not_ mean the `git+` in front, that one is normal because it's used to tell the browser/operating system to open it with a Git Application (for example the Github App on mobile)
i was confused about the `victorgarciaesgi@` in front of the github.com

**Comment by victorgarciaesgi** [comment]
You're right it's weird, i mean it's a valid url but not the cleanest aha, i'll look into it

**Comment by victorgarciaesgi** [comment]
sorry it took so long, it's fixed on `3.6.0`
